scout/parse/variant/variant.py

```diff
@@ -241,14 +242,16 @@ def parse_variant(
     set_rank_result(parsed_variant, variant, rank_results_header)
 
     ###################### Add SV specific annotations ######################
-    sv_frequencies = parse_sv_frequencies(variant)
-    for key in sv_frequencies:
-        parsed_variant["frequencies"][key] = sv_frequencies[key]
+    if parsed_variant.get("category") in ["sv", "cancer_sv"]:
```

Comment:
Consider setting the parsed_variant["frequencies"][key] directly in parse_sv_frequencies, This way you won't add complexity in this already complicated function

Reply:
I was about to say this could be a pydantic object instead, but I guess the more we simplify it the easier it would be to switch when we have a  need for that. 😄 Will fix!